net/nlbwmon/files/nlbwmon.hotplug

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+[ -n "$DEVICE" ] || exit 0
+
+[ "$ACTION" = ifup ] && /etc/init.d/nlbwmon enabled && /etc/init.d/nlbwmon reload
+
+logger -t nlbwmon "Reloading nlbwmon due to $ACTION of $INTERFACE ($DEVICE)"
```

Comment:
Should this not be better shown only if we did really an reload?

Reply:
Fixed to log a message only when reloading.